src/components/graph/Graph.types.ts

```diff
@@ -28,6 +28,7 @@ export interface IGraphConfigZoom {
   maxZoom: number;
   minZoom: number;
   initialZoom: number;
+  dragGraphEnabled?: boolean;
```

Comment:
To be consistent with `zoomByScroll` could you rename it `panByDrag`?

Reply:
thanks. updated.